frontend/pages/hosts/ManageHostsPage/constants.ts

```diff
@@ -9,30 +9,36 @@ export const HOST_SELECT_STATUSES = [
     disabled: false,
     label: "All hosts",
     value: "",
-    helpText: "All hosts that have been enrolled to Fleet.",
+    helpText: "All hosts added to Fleet.",
   },
   {
     disabled: false,
     label: "Online hosts",
     value: "online",
-    helpText: "Hosts that have recently checked in to Fleet.",
+    helpText: "Hosts that will respond to a live query.",
   },
   {
     disabled: false,
     label: "Offline hosts",
     value: "offline",
-    helpText: "Hosts that have not checked in to Fleet recently.",
+    helpText: "Hosts that won’t respond to a live query.",
   },
   {
     disabled: false,
     label: "Missing hosts",
     value: "missing",
-    helpText: "Hosts that have been offline for 30 days or more.",
+    helpText: "Hosts that have been offline for 10 days or more.",
```

Comment:
@noahtalerman The dashboard uses the `host_summary` API key called `missing_30_days_count` to display on the dashboard, it links to this dropdown being used which uses the `host` key called `missing`. Are we changing the backend to check for 10 days? If so, we need to change the key name and copy being used on the dashboard.

Reply:
Good catch!

Reply:
> Are we changing the backend to check for 10 days?

@RachelElysia no. The UI should inform the user that "Missing" means offline for _30_ days or more.

Great catch! This is a Figma bug. I updated the Figma page to say "Hosts that have been offline for 30 days or more."